authors/settings.py

```diff
@@ -133,11 +133,35 @@
 # called `INSTALLED_APPS`.
 AUTH_USER_MODEL = 'authentication.User'
 
-REST_FRAMEWORK = {
-    'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
-    'NON_FIELD_ERRORS_KEY': 'error',
+# REST_FRAMEWORK = {
+#     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
+#     'NON_FIELD_ERRORS_KEY': 'error',
+
+#     'DEFAULT_AUTHENTICATION_CLASSES': (
+#         'authors.apps.authentication.backends.JWTAuthentication',
+#     ),
+# }
+
+# REST_FRAMEWORK = {
+#    # When you enable API versioning, the request.version attribute will contain a string
+#    # that corresponds to the version requested in the incoming client request.
+#    'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.URLPathVersioning',
+#    'DEFAULT_AUTHENTICATION_CLASSES': (
+#        'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
+#    ),
+#    'DEFAULT_PERMISSION_CLASSES': (
+#        'rest_framework.permissions.IsAuthenticated',
+#     )
+# }
 
+REST_FRAMEWORK = {
```

Comment:
Please retain the authentication code as it was. 

I understand we only had this discussion right after you made these changes.

This way the team working on authentication will start on a clean slate 😃